.vscode/settings.json

```diff
@@ -3,5 +3,6 @@
   "prettier.singleQuote": true,
   "prettier.semi": false,
   "editor.tabSize": 2,
-  "editor.formatOnSave": true
+  "editor.formatOnSave": true,
+  "typescript.tsdk": "node_modules/typescript/lib"
```

Comment:
I don't mind this being here but versioning paths makes me nervous so is it necessary?

Reply:
Resolved as discussed here https://github.com/decentraland/decentraland-dapps/pull/25#discussion_r233849611